<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Nimer" failures="1" tests="6" name="Default test" time="25.051" errors="0" timestamp="2023-11-20T16:18:48 EET">
  <testcase classname="MyClass" name="CheckTheLanguage" time="0.073"/>
  <testcase classname="MyClass" name="DateCheck" time="0.006">
    <failure type="java.lang.AssertionError" message="Departure date mismatch expected [your_actual_departure_date_as_string] but found [21]">
      <![CDATA[java.lang.AssertionError: Departure date mismatch expected [your_actual_departure_date_as_string] but found [21]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.asserts.Assertion$10.doAssert(Assertion.java:176)
at org.testng.asserts.Assertion.executeAssert(Assertion.java:25)
at org.testng.asserts.Assertion.doAssert(Assertion.java:12)
at org.testng.asserts.Assertion.assertEquals(Assertion.java:172)
at MyClass.DateCheck(MyClass.java:142)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- DateCheck -->
  <testcase classname="MyClass" name="checkTheContactNumber" time="0.193"/>
  <testcase classname="MyClass" name="checkTheCurrency" time="0.052"/>
  <testcase classname="MyClass" name="checkingLoadPage" time="0.002"/>
  <testcase classname="MyClass" name="myTest" time="19.613"/>
  <testcase classname="MyClass" name="HotelTab" time="0.0">
    <ignored/>
  </testcase> <!-- HotelTab -->
</testsuite> <!-- Default test -->
